lib/steps.js

```diff
@@ -65,6 +73,40 @@ Given(/I start the interview at "([^"]+)"[ on ]?(.*)/, async (file_name, optiona
   if (!scope.page) {
     scope.page = await scope.browser.newPage();
     scope.page.setDefaultTimeout(120 * 1000)
+    // Files should get downloaded
+    scope.docs_dir = `downloads_${ scope.scenario_name }`
+    await scope.page._client.send('Page.setDownloadBehavior', {
+      behavior: 'allow',
+      downloadPath: path.resolve( scope.docs_dir ),  // Save in root of this project
+    });
+
+    // ---- EVENT LISTENERS ----
+    // From https://stackoverflow.com/a/51423688
```

Comment:
This block seems like an incantation that I don't entirely understand, but if you've tested and it works, then I'm ready to believe in this spell 🧙

Reply:
The sorcery of stackoverflow...